nntrainer/tensor/float_tensor.cpp

```diff
@@ -476,6 +476,60 @@ TensorV2 &FloatTensor::divide(TensorV2 const &m, TensorV2 &output) const {
   return output;
 }
 
+TensorV2 &FloatTensor::add_strided(TensorV2 const &input, TensorV2 &output,
+                                   const float beta) const {
+  NNTR_THROW_IF(getData() == nullptr, std::invalid_argument)
+    << getName() << " is not allocated";
+  NNTR_THROW_IF(input.getData<float>() == nullptr, std::invalid_argument)
+    << input.getName() << " is not allocated";
+  NNTR_THROW_IF(output.getData<float>() == nullptr, std::invalid_argument)
+    << output.getName() << " is not allocated";
+
+  if (strides[3] != 1 || input.getStrides()[3] != 1 ||
+      output.getStrides()[3] != 1 || beta != 0.0) {
```

Comment:
Are you really sure it's ok to use ```beta != 0.0``` with ```float beta```? 

Reference: https://stackoverflow.com/questions/4915462/how-should-i-do-floating-point-comparison

Reply:
Good point. Fixed :)